package-lock.json

```diff
@@ -9042,28 +9042,28 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "",
+          "resolved": false,
```

Comment:
Can you discard changes to the package-lock file?

Reply:
npm run boot update automatically every package lock. I can't install module without lerna update something, no?

Reply:
yeah but the changes on this specific file doesn't feel so good